__tests__/submissions.test.ts

```diff
@@ -164,19 +178,31 @@ describe('getSubmissionById', () => {
 
     beforeAll(async () => {
         ({ url } = await startStandaloneServer(server, {listen: { port: 0 }}))
-        await initiatilizeFirebaseInstance()
+        testEnv = await initializeTestEnvironment({
+            projectId: process.env.FIRESTORE_PROJECT_ID,
+            firestore: {
+                rules: fs.readFileSync('./firestore.rules', 'utf8')
+            }
+        })
```

Comment:
I'm wondering how we can refactor `testEnv` so it doesn't need to be defined for every test. 🤔